libr/crypto/p/crypto_aes_algo.h

```diff
@@ -266,7 +269,7 @@ static const ut32 U3[256] = { UT };
 #undef V
 #undef UT
 
-void aes_encrypt(RCryptoAESState*, ut8*, ut8*);
-void aes_decrypt(RCryptoAESState*, ut8*, ut8*);
-
+bool aes_ecb (RCryptoAESState *, ut8 *const, ut8 *, bool, const int);
```

Comment:
```suggestion
R_IPI bool aes_ecb(RCryptoAESState *, ut8 *const, ut8 *, bool, const int);
```

function definitions dont have a space before (
also, R_IPI is necessary to clarify the visibility